doc/source/tutorial-quickstart-scikitlearn.rst

```diff
@@ -45,41 +45,51 @@ However, before setting up the client and server, we will define all functionali
 * :code:`get_model_parameters()`
     * Returns the parameters of a :code:`sklearn` LogisticRegression model
 * :code:`set_model_params()`
-    * Sets the parameters of a :code:`sklean` LogisticRegression model
+    * Sets the parameters of a :code:`sklearn` LogisticRegression model
 * :code:`set_initial_params()`
     * Initializes the model parameters that the Flower server will ask for
-* :code:`load_mnist()`
-    * Loads the MNIST dataset using OpenML
-* :code:`shuffle()`
-    * Shuffles data and its label
-* :code:`partition()`
-    * Splits datasets into a number of partitions
 
 Please check out :code:`utils.py` `here <https://github.com/adap/flower/blob/main/examples/sklearn-logreg-mnist/utils.py>`_ for more details.
 The pre-defined functions are used in the :code:`client.py` and imported. The :code:`client.py` also requires to import several packages such as Flower and scikit-learn:
 
 .. code-block:: python
 
+  import argparse
   import warnings
-  import flwr as fl
-  import numpy as np
-
+  
   from sklearn.linear_model import LogisticRegression
   from sklearn.metrics import log_loss
-
+  
+  import flwr as fl
   import utils
+  from flwr_datasets import FederatedDataset
 
-
-We load the MNIST dataset from `OpenML <https://www.openml.org/search?type=data&sort=runs&id=554>`_, a popular image classification dataset of handwritten digits for machine learning. The utility :code:`utils.load_mnist()` downloads the training and test data. The training set is split afterwards into 10 partitions with :code:`utils.partition()`.
+Prior to local training, we need to load the MNIST dataset, a popular image classification dataset of handwritten digits for machine learning, and partition the dataset for FL. This can be conveniently achieved using Flower Datasets.
```

Comment:
```suggestion
Prior to local training, we need to load the MNIST dataset, a popular image classification dataset of handwritten digits for machine learning, and partition the dataset for FL. This can be conveniently achieved using `Flower Datasets <https://flower.ai/docs/datasets>`_.
```

Reply:
looks like the perfect place to add a link

Reply:
Good suggestion! I've added the link, thanks.